redundant-bmc/src/rbmc_tool.cpp

```diff
@@ -394,7 +382,7 @@ int main(int argc, char** argv)
     }
     else if (resetSibling)
     {
-        resetSiblingBMC();
+        ctx.spawn(resetSiblingBMC(ctx));
```

Comment:
I assume, we will wait to complete sibling reset.

Reply:
I noticed that `request_stop` is also triggered from `main`, and the context's `run` call will wait for all spawned tasks to complete, even after a stop is requested. This means `rbmctool` will finish any running tasks before stopping, and since `request_stop` prevents new tasks from being accepted and wait for running tasks.